Steps4Impact/NotificationsV2/NotificationsPermissionView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  NotificationsPermissionView.swift
+//  Steps4Impact
+//
+//  Created by Aalim Mulji on 10/25/20.
+//  Copyright © 2020 AKDN. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol NotificationPermissionCellDelegate: class {
+  func turnOnNotifictions()
+  func close()
+}
+
+struct NotificationPermissionCellContext: CellContext {
+  let identifier: String = NotificationPermissionCell.identifier
+  let title: String
+  let description: String
+  let disclosureText: String
+}
+
+class NotificationPermissionCell: ConfigurableTableViewCell {
+  static var identifier: String = "NotificationPermissionCell"
+  
+  private let cardView = CardViewV2()
+  private let disclosureView = CellDisclosureView()
+  
+  weak var delegate: NotificationPermissionCellDelegate?
+  
+  let headerLabel = UILabel(typography: .title)
+  let descriptionLabel = UILabel(typography: .smallRegular, color: Style.Colors.FoundationGrey.withAlphaComponent(0.5))
+  
+  let closeButton: UIImageView = {
```

Comment:
There is an extension on UIButton that takes an asset enum also if you want to use that.